commands/poll.js

```diff
@@ -211,20 +217,38 @@ module.exports = {
         const option = interaction.options.getString(`option-${i}`);
 
         if (option) {
+          const optionAlreadyExists = options.length > 0 ?
+            options.map((item) => item.option.trim().toLowerCase()).includes(option.trim().toLowerCase())
+            : false;
```

Comment:
Minor: It might be better to have `optionAlreadyExists` be `options.length > 1 && options.map((item) => item.option.trim().toLowerCase()).includes(option.trim().toLowerCase())`.

With this, if the first condition is already `false` (i.e., bot just got the first option), then it's automatically `false`.

Also, you can probably remove the "Already" part of the variable.